### PR TITLE
bounty re-upload

### PR DESCRIPTION
**Pull request by Jmimoni**

### CODE REVIEW

File: bounties/npm/cordova-serve/1/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Remote Code Execution",
+    "Author": {
+        "Username": "",
```
Comment (adam-nygate) [inline bounties/npm/cordova-serve/1/vulnerability.json:7]: Missing

File: bounties/npm/cordova-serve/1/vulnerability.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "Name": "Michele Romano"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "cordova-serve",
+        "URL": ""
```
Comment (adam-nygate) [inline bounties/npm/cordova-serve/1/vulnerability.json:13]: Missing

File: bounties/npm/cordova-serve/1/vulnerability.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    },
+    "CWEs": [
+        {
+            "ID": "77",
+            "Description": ""
```
Comment (adam-nygate) [inline bounties/npm/cordova-serve/1/vulnerability.json:18]: Missing

File: bounties/npm/cordova-serve/1/vulnerability.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "7.8"
+    },
+    "CVEs": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/cordova-serve/1/vulnerability.json:37]: Remove

File: bounties/npm/cordova-serve/1/vulnerability.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    ],
+    "Repository": {
+        "URL": "https://github.com/apache/cordova-serve",
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "Permalinks": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/cordova-serve/1/vulnerability.json:46]: Remove

File: bounties/npm/cordova-serve/1/vulnerability.json [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    ],
+    "References": [
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/apache/cordova-serve/blob/master/src/browser.js#L111"
+        },
+        {
+            "Description": "Permalink",
+            "URL": "https://github.com/apache/cordova-serve/blob/master/src/browser.js#L118"
+        }
```
Comment (adam-nygate) [inline bounties/npm/cordova-serve/1/vulnerability.json:56]: These need to be migrated to the Permalinks section

File: bounties/npm/dzzoffice/1/vulnerability.json [hunk 7]
```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Cross Site Scripting",
+    "Author": {
+        "Username": "",
```
Comment (adam-nygate) [inline bounties/npm/dzzoffice/1/vulnerability.json:7]: Missing

File: bounties/npm/dzzoffice/1/vulnerability.json [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+        "Name": "Michele Romano"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "dzzoffice",
+        "URL": ""
```
Comment (adam-nygate) [inline bounties/npm/dzzoffice/1/vulnerability.json:13]: Missing

File: bounties/npm/dzzoffice/1/vulnerability.json [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    "CWEs": [
+        {
+            "ID": "79",
+            "Description": ""
```
Comment (adam-nygate) [inline bounties/npm/dzzoffice/1/vulnerability.json:18]: Missing

File: bounties/npm/dzzoffice/1/vulnerability.json [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "6.7"
+    },
+    "CVEs": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/dzzoffice/1/vulnerability.json:37]: Remove

File: bounties/npm/dzzoffice/1/vulnerability.json [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+        ]
+    },
+    "Permalinks": [
+        ""
+    ],
+    "References": [
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/zyx0814/dzzoffice/blob/master/admin/setting/index.php#L200"
+        }
```
Comment (adam-nygate) [inline bounties/npm/dzzoffice/1/vulnerability.json:52]: Migrated to Permalinks

File: bounties/npm/dzzoffice/1/vulnerability.json [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+        ""
+    ],
+    "Repository": {
+        "URL": "https://github.com/zyx0814/dzzoffice/",
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "Permalinks": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/dzzoffice/1/vulnerability.json:46]: Remove

File: bounties/npm/launchpad/1/vulnerability.json [hunk 13]
```diff
@@ -0,0 +1,62 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Remote Code Execution",
+    "Author": {
+        "Username": "",
```
Comment (adam-nygate) [inline bounties/npm/launchpad/1/vulnerability.json:7]: Missing

File: bounties/npm/launchpad/1/vulnerability.json [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+        "Name": "Michele Romano"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "Launchpad",
+        "URL": ""
```
Comment (adam-nygate) [inline bounties/npm/launchpad/1/vulnerability.json:13]: Missing

File: bounties/npm/launchpad/1/vulnerability.json [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+    },
+    "CWEs": [
+        {
+            "ID": "77",
+            "Description": ""
```
Comment (adam-nygate) [inline bounties/npm/launchpad/1/vulnerability.json:18]: Missing

File: bounties/npm/launchpad/1/vulnerability.json [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "7.8"
+    },
+    "CVEs": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/launchpad/1/vulnerability.json:37]: Remove

File: bounties/npm/launchpad/1/vulnerability.json [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+    ],
+    "Repository": {
+        "URL": "https://github.com/bitovi/launchpad/",
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "Permalinks": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/launchpad/1/vulnerability.json:46]: Remove

File: bounties/npm/launchpad/1/vulnerability.json [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+            "URL": "https://github.com/bitovi/launchpad/blob/master/lib/local/instance.js#L12"
+        },
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/bitovi/launchpad/blob/master/lib/local/instance.js#L93"
+        },
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/bitovi/launchpad/blob/master/lib/local/instance.js#L97"
+        }
```
Comment (adam-nygate) [inline bounties/npm/launchpad/1/vulnerability.json:60]: Migrate to Permalinks

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 19]
```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Open Redirect",
+    "Author": {
+        "Username": "",
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:7]: Missing

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+        "Name": "Michele Romano"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "mod-auth-pubtkt",
+        "URL": ""
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:13]: Missing

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 21]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    "CWEs": [
+        {
+            "ID": "601",
+            "Description": ""
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:18]: Missing

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 22]
```diff
@@ -0,0 +1,54 @@
...
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "7.5"
+    },
+    "CVEs": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:37]: Remove

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 23]
```diff
@@ -0,0 +1,54 @@
...
+    ],
+    "Repository": {
+        "URL": "https://github.com/manuelkasper/mod_auth_pubtkt/",
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "Permalinks": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:46]: Remove

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 24]
```diff
@@ -0,0 +1,54 @@
...
+    ],
+    "References": [
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/manuelkasper/mod_auth_pubtkt/blob/master/php-login/login.php#L105"
+        }
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:52]: Migrate to permalinks

File: bounties/npm/strider-git/1/vulnerability.json [hunk 25]
```diff
@@ -0,0 +1,53 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Remote Code Execution",
+    "Author": {
+        "Username": "",
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:7]: Missing

File: bounties/npm/strider-git/1/vulnerability.json [hunk 26]
```diff
@@ -0,0 +1,53 @@
...
+        "Name": "Michele Romano"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "strider-git",
+        "URL": ""
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:13]: Missing

File: bounties/npm/strider-git/1/vulnerability.json [hunk 27]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    "CWEs": [
+        {
+            "ID": "77",
+            "Description": ""
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:18]: Missing

File: bounties/npm/strider-git/1/vulnerability.json [hunk 28]
```diff
@@ -0,0 +1,53 @@
...
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "7.8"
+    },
+    "CVEs": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:37]: Remove

File: bounties/npm/strider-git/1/vulnerability.json [hunk 29]
```diff
@@ -0,0 +1,53 @@
...
+            "JavaScript"
+        ]
+    },
+    "Permalinks": [
+        ""
+    ],
+    "References":   {
+        "Description": "GitHub Permalink",
+        "URL": "https://github.com/Strider-CD/strider-git/blob/master/lib/index.js#L151"
+    }
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:51]: Incorrect structure

Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:51]: Migrate to permalinks

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 30]
```diff
@@ -0,0 +1,53 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Arbitrary Code Execution",
+    "Author": {
+        "Username": "",
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:7]: Missing

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 31]
```diff
@@ -0,0 +1,53 @@
...
+        "Name": "Michele Romano"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "windows-edge",
+        "URL": ""
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:13]: Missing

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 32]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    "CWEs": [
+        {
+            "ID": "77",
+            "Description": ""
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:18]: Missing

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 33]
```diff
@@ -0,0 +1,53 @@
...
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "7.3"
+    },
+    "CVEs": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:37]: Remove

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 34]
```diff
@@ -0,0 +1,53 @@
...
+    ],
+    "Repository": {
+        "URL": "https://github.com/eugeneware/windows-edge",
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "Permalinks": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:46]: Remove

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 35]
```diff
@@ -0,0 +1,53 @@
...
+    ],
+    "References":   {
+        "Description": "GitHub Permalink",
+        "URL": "https://github.com/eugeneware/windows-edge/blob/master/index.js#L8"
+    }
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:51]: Incorrect structure

Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:51]: Migrate to permalinks

File: bounties/npm/curlrequest/2/vulnerability.json [hunk 36]
```diff
@@ -0,0 +1,52 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-05-07",
+    "AffectedVersionRange": "*",
+    "Summary": "Arbitraty File Read",
+    "Author": {
```
Comment (adam-nygate) [inline bounties/npm/curlrequest/2/vulnerability.json:6]: Missing

File: bounties/npm/curlrequest/2/vulnerability.json [hunk 37]
```diff
@@ -0,0 +1,52 @@
...
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "Permalinks": [],
+    "References": [
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/node-js-libs/curlrequest/blob/master/index.js#L239"
+        }
```
Comment (adam-nygate) [inline bounties/npm/curlrequest/2/vulnerability.json:50]: Migrate to Permalink

File: bounties/npm/curlrequest/2/vulnerability.json (hunk 36 above)
Comment (Jmimoni) [reply bounties/npm/curlrequest/2/vulnerability.json:6]: Author unknown

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json (hunk 20 above)
Comment (Jmimoni) [reply bounties/npm/mod-auth-pubtkt/1/vulnerability.json:13]: not available

File: bounties/npm/cordova-serve/1/vulnerability.json [hunk 38]
```diff
@@ -34,25 +34,25 @@
...
         {
-            "Description": "GitHub Permalink",
+            "Description": "Permalink 1",
             "URL": "https://github.com/apache/cordova-serve/blob/master/src/browser.js#L111"
         },
         {
-            "Description": "Permalink",
+            "Description": "Permalink 2",
             "URL": "https://github.com/apache/cordova-serve/blob/master/src/browser.js#L118"
         }
```
Comment (adam-nygate) [inline bounties/npm/cordova-serve/1/vulnerability.json]: Should be just strings

File: bounties/npm/curlrequest/2/vulnerability.json [hunk 39]
```diff
@@ -42,11 +42,13 @@
             "JavaScript"
         ]
     },
-    "Permalinks": [],
-    "References": [
+    "Permalinks": [
         {
             "Description": "GitHub Permalink",
             "URL": "https://github.com/node-js-libs/curlrequest/blob/master/index.js#L239"
         }
```
Comment (adam-nygate) [inline bounties/npm/curlrequest/2/vulnerability.json]: Should be just strings

File: bounties/npm/dzzoffice/1/vulnerability.json [hunk 40]
```diff
@@ -43,12 +43,12 @@
         ]
     },
     "Permalinks": [
-        ""
-    ],
-    "References": [
         {
             "Description": "GitHub Permalink",
             "URL": "https://github.com/zyx0814/dzzoffice/blob/master/admin/setting/index.php#L200"
         }
```
Comment (adam-nygate) [inline bounties/npm/dzzoffice/1/vulnerability.json]: Should be just strings

File: bounties/npm/launchpad/1/vulnerability.json [hunk 41]
```diff
@@ -43,20 +43,20 @@
...
         {
-            "Description": "GitHub Permalink",
+            "Description": "GitHub Permalink 2",
             "URL": "https://github.com/bitovi/launchpad/blob/master/lib/local/instance.js#L93"
         },
         {
-            "Description": "GitHub Permalink",
+            "Description": "GitHub Permalink 3",
             "URL": "https://github.com/bitovi/launchpad/blob/master/lib/local/instance.js#L97"
         }
```
Comment (adam-nygate) [inline bounties/npm/launchpad/1/vulnerability.json]: Should be just strings

File: bounties/npm/strider-git/1/vulnerability.json [hunk 42]
```diff
@@ -43,11 +43,12 @@
         ]
     },
     "Permalinks": [
-        ""
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/Strider-CD/strider-git/blob/master/lib/index.js#L151"
+        }
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:49]: Should be just strings

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 43]
```diff
@@ -43,11 +43,13 @@
         ]
     },
     "Permalinks": [
-        ""
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/eugeneware/windows-edge/blob/master/index.js#L8"
+        }
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:49]: Should be just strings

**Comment by adam-nygate** [comment]
Just pausing the merge on this as I think there's a bug in the enricher...

**Comment by Jmimoni** [comment]
Awesome

**Pull request merged**